extensions/wikia/Helios/Helios.setup.php

```diff
@@ -32,3 +32,4 @@
 $wgHooks['UserSaveSettings'][] = 'Wikia\\Helios\\User::onUserSave';
 $wgHooks['UserSaveOptions'][] = 'Wikia\\Helios\\User::onUserSave';
 $wgHooks['UserLogout'][] = 'Wikia\\Helios\\User::onUserLogout';
+$wgHooks['InvalidateToken'][] = 'Wikia\\Helios\\User::onUserInvalidateToken';
```

Comment:
I  don't think putting this in a hook has any advantages. Hooks are used in places when you need to plug-in some behaviour on a specific event. Here it should be an event called something like `UserPasswordChange` and it should be fired from within `User` class (`app/includes/User.php`).